whois-nrtm4/src/main/java/net/ripe/db/nrtm4/dao/WhoisDao.java

```diff
@@ -0,0 +1,41 @@
+package net.ripe.db.nrtm4.dao;
+
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.Map;
+
+
+@Service
+public class WhoisDao {
+
+    private final WhoisObjectRepository whoisObjectRepository;
+
+    WhoisDao(
+        final WhoisObjectRepository whoisObjectRepository
+    ) {
+        this.whoisObjectRepository = whoisObjectRepository;
+    }
+
+    @Transactional
+    public InitialSnapshotState getInitialSnapshotState() {
+        final int lastSerialId = whoisObjectRepository.findLastSerialId();
+        final List<ObjectData> objects = whoisObjectRepository.findLastObjects();
```

Comment:
transaction level is read committed, right? Will the last serial Id and getting last objects can be out of date?

Reply:
Duplicate commen